app/views/art_labels/show.html.erb

```diff
@@ -3,4 +3,60 @@
 Brewery: <%= @art_label.brewery %><br>
 Beer Description: <%= @art_label.beer_description %><br>
 Image: <%= @art_label.image_url %><br>
-Submitted by: <%= @art_label.user.username %><br>
+
+Submitted by: <%= @creator %><br>
+
+<h3>Reviews:</h3>
+<ul>
+  <% @reviews.each do |review| %>
+    <li>
+      <%= review.feels %>
+      <% if current_user && current_user.admin? %>
+       <%= button_to 'Delete', art_label_review_path(@art_label, review), data: { confirm: "Are you sure?" }, method: :delete %>
+     <% end %>
+    </li>
+  <% end %>
+</ul>
+
+<h3>Write a Review!</h3>
+
+<%= form_for([@art_label, @review]) do |f| %>
+
+  <%= f.label :feels %>
+  <%= f.text_field :feels %><br /><br />
+
+  <%= f.label :intoxication_level, 'Intoxication Level:' %>
+  <%= f.collection_select(:intoxication_level, [[1, 'Buzzed'],[2, 'More Buzzed'],[3, 'Drunk'],[4, 'More Drunk'],[5, 'Wasted']], :first, :last) %>  <br /><br />
+
+  <%= f.label :joy, 'Joy:' %>
+  <%= f.collection_radio_buttons(:joy, [[1],[2],[3],[4],[5]], :first, :first) %><br /><br />
```

Comment:
Nice radio buttons!

Reply:
Yeah, we'll take credit for this one. :)

Reply:
But really Al and Bobby did this one.